### PR TITLE
Add Posthog

### PR DESCRIPTION
**Pull request by toolmantim**
Adds PostHog tracking. The required environment variable values have already been added.

### CODE REVIEW

**Comment by toolmantim** [comment]
I just double checked this again, and looks good to go! Gonna merge 🚁

File: app/models/posthog.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Posthog
+  # Default to the free, hosted Posthog account which we send all dev data
+  URL = ENV.fetch("POSTHOG_URL", "https://app.posthog.com").freeze
+  API_KEY = ENV.fetch("POSTHOG_API_KEY") { "fPfkF4tU2OBcVBZhcshmzC7_VglTw078wuqD3tl01ic" }.freeze
```
Comment (sj26) [inline app/models/posthog.rb:6]: (p.s. Is this a legit key for a dev instance? Do we all have access to it, if so?)

Comment (sj26) [reply app/models/posthog.rb:6]: (keeping in mind this is a public repo)

**Comment by plaindocs** [comment]
Awesomeness

File: app/models/posthog.rb (hunk 1 above)
Comment (toolmantim) [reply app/models/posthog.rb:6]: Yep, these are public keys (they're exposed in the page source)

**Pull request merged**